app/api/v1/mediator/manifest.py

```diff
@@ -1,53 +1,142 @@
-from typing import Any, Optional
-
-from fastapi import APIRouter, Body, Depends, status
+from typing import Any, Optional, Tuple
+import sys
+from fastapi import APIRouter, Body, Depends, HTTPException, status, Response
 
+from electionguard.group import ElementModQ, hex_to_q
 from electionguard.manifest import Manifest
 from electionguard.schema import validate_json_schema
+from electionguard.utils import get_optional
 
 from app.core.schema import get_description_schema
 
-from ..models import ValidateManifestRequest, BaseValidationResponse, ResponseStatus
+from ....core.repository import get_repository, DataCollection
+from ..models import (
+    ManifestQueryResponse,
+    ManifestSubmitResponse,
+    ValidateManifestRequest,
+    BaseValidationResponse,
+    ResponseStatus,
+)
 from ..tags import MANIFEST
 
 router = APIRouter()
 
+# TODO: multi-tenancy
+CLIENT_ID = "electionguard-default-client-id"
```

Comment:
i figure this stub is ok for now.  we'll need a way to handle multi-tenancy at some point but I'd be inclined to make an issue for this and handle it later once I hook everything up in the election context PR

Reply:
my general theory here is that when we have authorization then all of the election state stuff is tied to a tenant, and then all of the election-specific stuff is tied to that election state, so you can't query the ballot stuff if you can't resolve the election state